.github/workflows/generate-desktop-targets.yml

```diff
@@ -14,7 +14,7 @@ on:
   workflow_dispatch:
 
 env:
-  FLEET_DESKTOP_VERSION: 0.0.2
+  FLEET_DESKTOP_VERSION: 0.0.13
```

Comment:
Should we? They are different components (e.g. you could release an update to Desktop only)

If this is an issue with the displayed version, another option is to display both Orbit and Desktop versions.